doc/ref_model/chapters/chapter03.md

```diff
@@ -185,4 +185,30 @@ Networking, alongside Compute and Storage, is an integral part of the Cloud Infr
 
 Normalization of the integration reference points between different layers of the Cloud Infrastructure architecture is one of the main concerns. In the networking context the primary focus is directed on the packet flow and control flow interfaces between the virtual reosurces (referred to as Software (SW) Virtualisation Layer) and physical resources (referred to as Hardware (HW) Infrastructure Layer), as well as on related integration into the various MANO reference points (hardware/network infrastructure management, orchestration). The identification of these two different layers (SW Virtualisation Layer and HW Infrastructure Layer) remains in alignment with the separation of resources into virtual and physical resources, generally used in this document, see e.g. Figure 3-1. The importance of understanding the seperation of concerns between SW Virtualisation Layer and HW Infrastructure Layer is important because without it, the cardinality of having multiple CaaS and IaaS instances executing on their own private virtual resources from the single shared HW Infrastructure Layer cannot be expressed into separate administrative domains.
 
+<p align="center"><img src="../figures/RM-Chapter4-Network-Model.png" alt="Network Model" Title="Network Model" width="65%"/></p>
+<p align="center"><b>Figure 3-5:</b> Network Model</p>
 
+Principles that should be followed during development and definition of Networking scopes for Reference Model, Reference Architectures and down to Reference Implementations and Reference Conformance test suites under CNTT:
+
+• Abstraction: A standardized layer between Virtualisation Layer domains and the Networking Hardware Layer resource pool hiding the implementation details of the Networking Hardware.
+
+    Note: Abstraction may be deployed in many different ways e.g. depending on target use case requirements, workload characteristics, different algorithm implementions of pipeline stages and available platforms. This includes Network Functions running on server nodes with or without programmable hardware acceleration, or networking support functions running on programmable network switches in the network.
```

Comment:
IMO, we cannot deploy "abstraction." The "network abstraction layer" (please replace with whatever term is decided) needs to be workload agnostic (see "Agnosticity) and by definition implementation and platform agnostic. Also, if we are asking for "standardized" then why would we have "different"?

"Note: The "network abstraction layer" supports, for example, physical resources with or without programmable hardware acceleration, or programmable network switches."

Reply:
@pgoyal01, text updated

Reply:
@walterkozlowski Thanks. 

The following still things dependent upon :details" that should be hideen: " In deployment phases this principle may be applied in many different ways e.g. depending on target use case requirements, workload characteristics, different algorithm implementions of pipeline stages and available platforms." 

Typo: " implementions" --> " implementations"